hokusai/services/deployment.py

```diff
@@ -46,7 +46,7 @@ def update(self, tag, constraint, git_remote, resolve_tag_sha1=True):
         print_green("Pushing deployment tags to %s..." % remote)
         shout("git tag -f %s" % self.context, print_output=True)
         shout("git tag %s" % deployment_tag, print_output=True)
-        shout("git push --force %s --tags" % remote, print_output=True)
+        shout("git push --force %s --tags --no-verify" % remote, print_output=True)
```

Comment:
I was actually thinking to change this to...

```
shout("git push -f --no-verify %s refs/tags/%s" % (remote, self.context), print_output=True)
shout("git push --no-verify %s refs/tags/%s" % (remote, deployment_tag), print_output=True)
```
... so as to only make sure to only push these tags, rather than any others you may have lying around locally... wdyt?

Reply:
Not sure what the `self.context` change is about, but a I assume you mean just the removal of `-f`, right? If so, that makes sense to me 👍

Reply:
`self.context` is here either `"staging"` or `"production"`, so this is pushing this pointer tag as well as the deployment timestamp tag only, rather than `git push --tags` which will push all local tags to the remote

Reply:
Ah gotcha. Yeah sounds good 👍